### ISSUE TITLE
samuelattard.com outbound connection (call home) never stops

### ISSUE THREAD

**Issue by jwmann** [MarshallOfSound/Google-Play-Music-Desktop-Player-UNOFFICIAL-#597]
I use Little Snitch to control my outbound connections.
When I launch the app. It attempts to connect to the server samuelattard.com

I understand that this is the main developer's personal website / server. I'm not 100% sure why/what reason it wants to connect for and I have nothing against simple 'phone home' checks for apps. However, I would like to personally deny this outbound connection. The same as I disallow outbound tracking connections from Google.

This is where in lies the problem.
The outbound connection for samuelattard.com has no _max tries count_ or even a _timeout_ so as long as I'm _not_ playing any music from the app, it will constantly attempt to connect causing _massive_ CPU spikes.

![screen shot 2016-03-14 at 21 07 23](https://cloud.githubusercontent.com/assets/436270/13764811/cfc51ef6-ea29-11e5-844b-cf2163683afb.png)
OS X 10.11.3
Version: 3.0.1

I'm not saying to completely take out this outbound connection (I honestly don't know why it's necessary to be honest but that's another subject). What I am suggesting is to at least implement a counter for a max_connections or a 30 second timeout or even both would ideal.

Let me know what you think.

**Comment by MarshallOfSound** [comment]
That connection is part of the mini player, basically there is a "default" image for when music isn't playing.  I'll probably move it into source control and package it with the release.

The unlimited retries is a side effect of how the default image works.  It detects an error fetching an image and then attempts to fetch the default image, if the default image fails it will of course try again :+1: 

TLDR: Fair enough, I'll move it to a local file

**Comment by MarshallOfSound** [comment]
So turns out that I got this bit wrong and i can't distribute the file locally because it is part of the separate mini player library.  I'll implement a max-retries there I guess?  But you really shouldn't be blocking an image :-1:

**Comment by jwmann** [comment]
This is still a problem
http://jwm.d.pr/nfTMe7/5tFtUmPq
